pkg/blockstorage/awsebs/awsebs.go

```diff
@@ -76,6 +76,9 @@ func NewProvider(config map[string]string) (blockstorage.Provider, error) {
 
 // newEC2Client returns ec2 client struct.
 func newEC2Client(awsRegion string, config *aws.Config, role string) (*EC2, error) {
+	if config == nil {
+		config = aws.NewConfig()
+	}
```

Comment:
Will the default config have valid `Credentials` set? Is it going to affect the flow in any way?

Reply:
Should we instead return an error if the config is nil and make the caller get the config?